styles/hydrogen.less

```diff
@@ -43,7 +42,7 @@
                 line-height: 1.5em;
                 text-align: center;
                 box-shadow: 0 0 2px 0px rgba(0,0,0,0.5);
-                margin: 0;
+                margin: -50% 0 0 -50%;
```

Comment:
Hmm but then it will look odd. I've got another idea, I'll try that later if it works.